recipes/calceph/bld.bat

```diff
@@ -0,0 +1,3 @@
+nmake /f Makefile.vc
+
+nmake /f Makefile.vc install DESTDIR=%LIBRARY_PREFIX%
```

Comment:
```suggestion
nmake /f Makefile.vc install DESTDIR=%LIBRARY_PREFIX%

rm %LIBRARY_PREFIX%\libexec
```
Is there a shared library option for the Windows build? You're only shipping static libs.

Same comment as unix about MATLAB mex files.

Reply:
As far as I can see, the build system on Windows supports only static libraries.